src/actionTypes.js

```diff
@@ -344,6 +344,9 @@ type EventUpdateMessageAction = {|
   type: typeof EVENT_UPDATE_MESSAGE,
   edit_timestamp: number,
   message_id: number,
+  message_ids: number[],
+  stream_id: number,
+  new_stream_id?: number,
```

Comment:
Let's accompany this kind of change by having a comment at the top of the event link to the relevant bit of docs:
https://zulip.com/api/get-events#update_message
That helps a lot in reviewing a change here, or in comparing in the future to the possibly-by-then-further-changed API. It also makes it easy to look up what each of these properties is documented as being supposed to mean.

We mostly don't have those now (in fact it looks like there's just one such example, `RestartEvent` in `eventTypes.js` -- which not coincidentally looks like the latest event we've added to our types), mainly because the bulk of these type definitions were written down in the reverse-engineering era, when there were no API docs to link to.